src/database/food.go

```diff
@@ -0,0 +1,79 @@
+package database
+
+import (
+	"foodloop/src/models"
+	"strings"
+)
+
+func GenerateFoods(tags []string) ([]models.Food, error) {
```

Comment:
Commentary: no logic change. this is just cut and pasted from `foodlist.go` file. and renamed from `GenerateFoodlist`.